mantidproject/mantid
# Workbench: ISIS Reflectometry C++ GUI

**Description of work.**
This PR will allow users to access the reflectometry GUI from Workbench.

**To test:**
- Open Workbench
- Open Interfaces->Reflectometry->ISIS Reflectometry
- Run through the manual tests.
- Try and break the GUI running through random button clicking and the like.
- Check all buttons have a hint when hovered over.



Fixes #25383  




*Still need to add release notes*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
